vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -17,6 +17,7 @@ def ems_inv_to_hashes(inventory)
       get_pods(inventory)
       get_endpoints(inventory)
       get_services(inventory)
+      get_namespaces(inventory)
```

Comment:
Move this after:

    get_nodes(inventory)

Parsing namespaces is the first thing you'll need to create relationships.